scengen/generator.py

```diff
@@ -77,6 +84,22 @@ def generate_scenario(options: dict) -> None:
     write_yaml(scenario, options["scenario_path"])
 
 
+def _get_number_of_agents_to_create(agent_count: Union[List[Any], Any], options: dict) -> int:
+    """
+    Returns an integer number from field `agent_count`
+    Accepts float values by rounding to integer, but raises warning to notify user about wrong type, in all other cases an Error is raised
```

Comment:
Change last port of doctring to
"...for any other datatype an Error is raised"

Reply:
Done